src/Platform.Handlers/src/Xamarin.Platform.Handlers/Platform/Android/SwitchExtensions.cs

```diff
@@ -0,0 +1,254 @@
+﻿using Android.Graphics.Drawables;
+using Xamarin.Forms;
+using ASwitch = AndroidX.AppCompat.Widget.SwitchCompat;
+using AAttribute = Android.Resource.Attribute;
+using Android.Content.Res;
+using AColor = Android.Graphics.Color;
+using System;
+using APorterDuff = Android.Graphics.PorterDuff;
+using AndroidX.Core.Graphics.Drawable;
+using AndroidX.AppCompat.Widget;
+using Android.Content;
+using Android.Util;
+
+namespace Xamarin.Platform
+{
+	public static class SwitchExtensions
+	{
+		static ColorTrackingColorStateList? mSwitchTrackStateList;
+		// taken from android sourcec ccode
+		static ColorTrackingColorStateList getSwitchTrackColorStateList(this Context context)
```

Comment:
@mattleibow @jsuarezruiz I think we'll need some version of this code regardless if we use ColorStateLists or not

Unless you all can find a better approach for setting the SwitchTrack Drawable back to `Color.Default`